common/lib/driver-definitions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluidframework/driver-definitions",
-  "version": "0.43.0",
+  "version": "0.43.1",
```

Comment:
Why did this do a patch bump...?

Reply:
Ohhh probably this:
* There were changes in main to driver-definitions in the last few weeks
* Client packages were updated to state their dependency on those changes
* Now that we cut the branch and releasing d-d 0.43.0 which contains those changes, we can update client packages to depend on that
* And further driver-definitions changes in this release branch will be part of 0.43.1